src/audio/base_fw.c

```diff
@@ -19,7 +19,7 @@
 #include <sof/lib_manager.h>
 #include <rtos/clk.h>
 #include <rtos/init.h>
-#include <platform/lib/clk.h>
+
```

Comment:
Hmm, I thought I had a point in adding this empty lne for grouping, but now I realize I really don't. Will fix in next version.